HARK/ConsumptionSaving/ConsAggShockModel.py

```diff
@@ -550,11 +550,325 @@ def getMrkvNow(self):  # This function exists to be overwritten in StickyE model
         return self.MrkvNow*np.ones(self.AgentCount, dtype=int)
 
 
+init_KS_agents = {
+    'T_cycle' : 1,
+    'DiscFac' : 0.99,
+    'CRRA' : 1.0,
+    'LbrInd' : 1.,
+    'aMin' : 0.001,
+    'aMax' : 50.,
+    'aCount' : 32,
+    'aNestFac' : 2,
+    'MgridBase' : np.array([0.1,0.3,0.6,0.8,0.9,0.95,0.98,1.0,1.02,1.05,1.1,1.2,1.6,2.0,3.0]),
+    'AgentCount' : 5000
+    }
+    
+class KrusellSmithType(AgentType):
+    '''
+    A class for representing agents in the seminal Krusell-Smith (1998) model from
+    the paper "Income and Wealth Heterogeneity in the Macroeconomy".  All default
+    parameters have been set to match those in the paper, but the equilibrium object
+    is perceptions of aggregate assets as a function of aggregate market resources
+    in each macroeconomic state (bad=0, good=1), rather than aggregate capital as
+    a function of previous aggregate capital.  This choice was made so that some
+    of the code from HARK's other HA-macro models can be used.
+    '''
+    
+    def __init__(self, **kwds):
+        '''
+        Make a new instance of the Krusell-Smith type.
+        '''
+        params = init_KS_agents.copy()
+        params.update(kwds)
+
+        AgentType.__init__(self, pseudo_terminal=False, **params)
+
+        # Add consumer-type specific objects
+        self.time_vary = []
+        self.time_inv = ['DiscFac', 'CRRA',]
+        self.poststate_vars = ['aNow','EmpNow']
+        self.solveOnePeriod = solveKrusellSmith
+        self.update()
+        
+    def preSolve(self):
+        self.update()
+        self.preComputeArrays()
+        
+    def update(self):
+        '''
+        Construct objects used during solution from primitive parameters.
+        '''
+        self.makeGrid()
+        self.updateSolutionTerminal()
+        
+    def getEconomyData(self, Economy):
```

Comment:
Should be lower case -- see how the GitHub syntax highlighting misformats this as a class name because it is capitalized.

I see this is common to other uses of getEconomyData though.